tests/integ/test_dataframe.py

```diff
@@ -416,6 +425,48 @@ def process(self, n: int) -> int:
     Utils.check_answer(df.select(df.a, table_func(df.b, lit(" "))), expected_result)
 
 
+def test_generator_table_function(session):
+    expected_result = [Row(127, 3), Row(-128, 3), Row(-127, 3)]
+
+    # works with timelimit
+    df = session.generator(seq1(1), uniform(1, 10, 2), rowcount=132).limit(
```

Comment:
Should we add order by before getting to limit, or is the result of `generator` already sorted? This might be flaky as it just picks three random rows that happens to be at the offset position.

Reply:
great suggestion. added `order_by`